restli-tools/src/main/java/com/linkedin/restli/tools/annotation/SchemaAnnotationHandlerUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.linkedin.restli.tools.annotation;
+
+import com.linkedin.data.schema.annotation.SchemaAnnotationHandler;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.StringTokenizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.linkedin.data.schema.resolver.FileDataSchemaResolver.*;
+
+
+/**
+ * Schema Annotation Handler utility
+ */
+public class SchemaAnnotationHandlerUtil
```

Comment:
Why is this duplicated? Can't you just use SchemaAnnotationHandlerClassUtil?

Reply:
This utility is getting SchemaAnnotationHandler list, it is in the restli-tools module. So that the SchemaAnnoationValidatorCmdLineApp could use SchemeAnnotationHandler objects. SchemaAnnotationHandlerClassUtil is in gradle-plugin module, it is used to get handler class names(String). Based on the current structure of pegasus gradle tasks, we cannot pass the Object(SchemaAnnotationHandler) from gradle-plugin task to the restli-tools main class, correct me if I am wrong.

Reply:
Actually, I am not using schemaAnnotationHandler for this validation. 
I think I'd better remove all those 2 Utils and keep the code as before, because it is unrelated to this change.